src/main/java/com/photolocos/enterprise/dto/LocationDTO.java

```diff
@@ -25,111 +25,18 @@
     @GeneratedValue(strategy= GenerationType.AUTO)
     private int id;
 
-    private String longitude;
-    private String latitude;
+    private double longitude;
+    private double latitude;
     private String country;
     private String state;
     private String city;
     private String area;
     private String description;
-
-
     private int locationId;
 
-    /*
-     *
-     * Getters and Setters
-     *
-     */
-
-    /**
-     *
-     * Gets the longitude of a location.
-     * @return the longitude of the location.
-     */
-    public String getLongitude() {
-        return longitude;
-    }
-
-    /**
-     *
-     * @param longitude, set the longitude of a location.
-     */
-    public void setLongitude(String longitude) {
-        this.longitude = longitude;
-    }
-
-    /**
-     *
-     * Gets the latitude of a location.
-     * @return the latitude of the location.
-     */
-    public String getLatitude() {
-        return latitude;
-    }
-
-    /**
-     *
-     * @param latitude, set the latitude of a location.
-     */
-    public void setLatitude(String latitude) {
-        this.latitude = latitude;
-    }
-
-    public String getCountry() {
-        return country;
-    }
-
-    public void setCountry(String country) {
-        this.country = country;
+    public boolean equals(LocationDTO location) {
+        return (this.getState().equals(location.getState()) && this.getCity().equals(location.getCity()))
+                || (this.getLatitude() == location.getLatitude() && this.getLongitude() == location.getLongitude())
+                || (this.getArea().equals(location.getArea()));
     }
```

Comment:
Created an equals method in LocationDTO for checking if two locations are equal.

Reply:
nice.